temporal101/README.md

```diff
@@ -76,3 +78,46 @@ You can examine the workflow's execution in a web interface, too. In a
 browser, open `localhost:8233` and click into the workflow you just ran.
 If you made a mistake in the inputs, this will make it easier to
 diagnose than reading the log output of the worker in the terminal.
+
+## Exercise 2: Start a workflow programmatically
+
+Ensure that you're running a Temporal server.
+
+### Part A: Review the client logic
+
+Read through `Exercise2.hs`. Compare its `main` function to that of
+`Exercise1.hs`.
+
+### Part B: Launch the worker and execute the workflow
+
+Run your worker from exercise 1 again:
+
+```bash
+$ cabal run temporal101:exercise1
+```
+
+In another terminal (in the nix shell), build and run exercise 2:
+
+```bash
+$ cabal run temporal101:exercise2
+```
+
+This will hang. Don't panic, leave it running.
```

Comment:
we could add a little logging here if the worker code doesn't make it clear that it's done something